templates/nav.hbs

```diff
@@ -20,8 +21,12 @@
   {{/each}}
 
   <button name="drawer" class="btn-icon nav__btn nav__drawer-btn js-nav-drawer-btn is-position-{{_config._drawer._position}}" data-event="toggleDrawer" aria-expanded="false" aria-label="{{_globals._accessibility._ariaLabels.navigationDrawer}}" data-order="{{_globals._extensions._drawer._navOrder}}" data-tooltip-id="drawer">
```

Comment:
Does this not become `btn-text` if there is no icon?

Reply:
Technically, it should yes. But I have taken the approach of leaving as is for the time being to reduce potential issues. 

If we were to change the button class to `btn-text` in the navigation and have the theme set up to have a different font for buttons than body, then we could run into an issue whereby the fonts are different between buttons in the same space.

<img width="367" alt="Screenshot 2024-02-19 at 12 29 34" src="https://github.com/adaptlearning/adapt-contrib-core/assets/11569678/04f07065-653c-4bb7-a284-e35b2301a885">